samples/src/main/java/terminal/TerminalJavaClient.java

```diff
@@ -162,8 +163,8 @@ public void execute() {
         @Override
         public void gatherInput() {
             LineReader reader = LineReaderBuilder.builder().build();
-
             input.setAmount(Long.parseLong(reader.readLine("\nAmount (in minor units, only digits): ")));
+            input.setShowTips(Boolean.parseBoolean(reader.readLine("\n Do you want to enable Terminal to show Tips (true/false): ")));
```

Comment:
I would use a different reader instance here, to handle validation in the reader. E.g.
```
LineReaderBuilder.builder()
                .completer(new StringsCompleter("true", "false"))
                .build();
```